package.json

```diff
@@ -1,16 +1,45 @@
 {
   "name": "fold",
-  "version": "0.12.0",
+  "version": "0.13.0",
   "description": "FOLD file format for origami models, crease patterns, etc.",
-  "main": "lib/index.js",
+  "main": "dist/index.civet.js",
   "bin": {
     "fold-convert": "bin/fold-convert.js"
   },
   "scripts": {
-    "test": "npm run coffee && jest",
-    "prepare": "npm run coffee && npm run dist",
-    "coffee": "coffee --no-header --bare -o lib -c src",
-    "dist": "browserify -t coffeeify --extension=.coffee -r ./src/index.coffee:fold -o dist/fold.js"
+    "test": "vitest",
+    "build": "vite build",
+    "dev": "vite dev"
+  },
+  "exports": {
+    ".": {
+      "import": "./dist/index.civet.js",
+      "require": "./dist/index.civet.mjs"
+    },
+    "./convert": {
+      "import": "./dist/convert.civet.js",
+      "require": "./dist/convert.civet.mjs"
+    },
+    "./file": {
+      "import": "./dist/file.civet.js",
+      "require": "./dist/file.civet.mjs"
+    },
+    "./filter": {
+      "import": "./dist/filter.civet.js",
+      "require": "./dist/filter.civet.mjs"
+    },
+    "./geom": {
+      "import": "./dist/geom.civet.js",
+      "require": "./dist/geom.civet.mjs"
+    },
+    "./viewer": {
+      "import": "./dist/viewer.civet.js",
+      "require": "./dist/viewer.civet.mjs"
+    },
+    "./oripa": {
+      "import": "./dist/oripa.civet.js",
+      "require": "./dist/oripa.civet.mjs"
+    }
```

Comment:
How do we expose `types.civet`?

Reply:
We aren't generating dts files here yet, but once we should just be able to add `types.civet` as another entrypoint in vite config